docs/user-guide/node-selection/index.md

```diff
@@ -198,3 +198,215 @@ must be satisfied for the pod to schedule onto a node.
 
 For more information on inter-pod affinity/anti-affinity, see the design doc
 [here](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/podaffinity.md).
+
+## Taints and tolerations (beta feature)
+
+### Specifying taints and tolerations
```

Comment:
Delete this heading; the paragraphs after explain what Taints and Tolerations are, not how to set them.